cli/crates/server/src/errors.rs

```diff
@@ -17,8 +17,8 @@ use crate::udf_builder::JavaScriptPackageManager;
 #[derive(Error, Debug)]
 pub enum JavascriptPackageManagerComamndError {
     /// returned if npm/pnpm/yarn cannot be found
-    #[error("could not find {0}: {1}")]
-    NotFound(JavaScriptPackageManager, which::Error),
+    #[error("could not find {0:?}: {1}")]
+    NotFound(Option<JavaScriptPackageManager>, String),
```

Comment:
Does this error get shown to users?  `None` or `Some(Npm)` don't seem like very good things to appear in user facing error messages